src/decorators/user-roles.decorators.ts

```diff
@@ -1,11 +1,13 @@
-import { createParamDecorator, PipeTransform } from '@nestjs/common';
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 
 /**
  * Access the user roles from the request object i.e `req.user.roles`.
  *
  * You can pass an optional property key to the decorator to get it from the user object
  * e.g `@UserRoles('premissions')` will return the `req.user.premissions` instead.
```

Comment:
typo

Reply:
Yeah, actually PipeTransform has not being used.